tests/e2e/main_test.go

```diff
@@ -277,3 +277,42 @@ func TestFailedRuleEvaluations(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestGrafana(t *testing.T){
+	kClient := promClient.kubeClient
+	grafanaScale, err := kClient.AppsV1().Deployments("monitoring").GetScale(context.Background(), "grafana", metav1.GetOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	
+	// Loop for 5 mins
+	for x := 0; x < 10; x++ {
```

Comment:
For the polling, we should use `wait.Poll` instead of doing it manually. There is an example in the test above if you want to have a look.